### PR TITLE
refactor: add new line to end of all Kotlin files

### PR DESCRIPTION
**Pull request by mayokunthefirst**
Fixes issue #235

### CODE REVIEW

File: coroutines-interop/src/nativeCommonMain/kotlin/com/badoo/reaktive/coroutinesinterop/LaunchCoroutine.kt [hunk 1]
```diff
@@ -14,7 +15,7 @@ internal actual inline fun <T> launchCoroutine(
     crossinline onError: (Throwable) -> Unit,
     crossinline block: suspend CoroutineScope.() -> T
 ) {
-    val disposable = Disposable()
+    val disposable = disposable()
```
Comment (arkivanov) [inline coroutines-interop/src/nativeCommonMain/kotlin/com/badoo/reaktive/coroutinesinterop/LaunchCoroutine.kt]: Why do you have such changes?

Comment (CherryPerry) [reply coroutines-interop/src/nativeCommonMain/kotlin/com/badoo/reaktive/coroutinesinterop/LaunchCoroutine.kt]: Looks like in `10dad76` he did mistake in branch merging, and reverted it in `bcc830f`. So previous implementation is kept and branch is merged.

Looks like it is required to checkout `e7176b5`, merge `master` again and force push branch.

**Comment by arkivanov** [comment]
Hey @mayokunthefirst I have fixed and force-pushed your branch, fortunately I had write access. Now changes look fine. @CherryPerry please review whenever you have time.

**Comment by CherryPerry** [comment]
@arkivanov Can you do squash merge via Github?

File: coroutines-interop/src/nativeCommonMain/kotlin/com/badoo/reaktive/coroutinesinterop/LaunchCoroutine.kt (hunk 1 above)
Comment (mayokunthefirst) [reply coroutines-interop/src/nativeCommonMain/kotlin/com/badoo/reaktive/coroutinesinterop/LaunchCoroutine.kt]: > Why do you have such changes?

I think I made a mistake when merging.

**Comment by mayokunthefirst** [comment]
> Hey @mayokunthefirst I have fixed and force-pushed your branch, fortunately I had write access. Now changes look fine. @CherryPerry please review whenever you have time.

Hello @arkivanov, thank you!

**Pull request merged**